### DISCUSSION TITLE
react-native-gesture-handler 관련 에러

### DISCUSSION THREAD

**Discussion by imdangun** [velopert/dealing-with-react-native#25] [버그]
p291 를 실행했더니, 다음 에러가 발생하네요. 해결책이 뭘까요?

BUNDLE  ./index.js

 WARN  If you want to use Reanimated 2 then go through our installation steps https://docs.swmansion.com/react-native-reanimated/docs/installation
 LOG  Running "rn" with {"rootTag":41}
 WARN  [react-native-gesture-handler] Seems like you're using an old API with gesture components, check out new Gestures system!
PanGestureHandler@http://10.0.2.2:8081/index.bundle?platform=android&dev=true&minify=false&app=com.rn&modulesOnly=false&runModule=true:131874:38
PanGestureHandler@http://10.0.2.2:8081/index.bundle?platform=android&dev=true&minify=false&app=com.rn&modulesOnly=false&runModule=true:131159:34
DrawerView@http://10.0.2.2:8081/index.bundle?platform=android&dev=true&minify=false&app=com.rn&modulesOnly=false&runModule=true:138620:36
DrawerViewBase@http://10.0.2.2:8081/index.bundle?platform=android&dev=true&minify=false&app=com.rn&modulesOnly=false&runModule=true:110349:22
RNGestureHandlerRootView
GestureHandlerRootView
RNCSafeAreaProvider
SafeAreaProvider@http://10.0.2.2:8081/index.bundle?platform=android&dev=true&minify=false&app=com.rn&modulesOnly=false&runModule=true:119617:24
SafeAreaProviderCompat@http://10.0.2.2:8081/index.bundle?platform=android&dev=true&minify=false&app=com.rn&modulesOnly=false&runModule=true:130835:24
DrawerView@http://10.0.2.2:8081/index.bundle?platform=android&dev=true&minify=false&app=com.rn&modulesOnly=false&runModule=true:110621:27
DrawerNavigator@http://10.0.2.2:8081/index.bundle?platform=android&dev=true&minify=false&app=com.rn&modulesOnly=false&runModule=true:110187:32
EnsureSingleNavigator@http://10.0.2.2:8081/index.bundle?platform=android&dev=true&minify=false&app=com.rn&modulesOnly=false&runModule=true:123137:24
BaseNavigationContainer@http://10.0.2.2:8081/index.bundle?platform=android&dev=true&minify=false&app=com.rn&modulesOnly=false&runModule=true:122646:28
ThemeProvider@http://10.0.2.2:8081/index.bundle?platform=android&dev=true&minify=false&app=com.rn&modulesOnly=false&runModule=true:128807:21
NavigationContainerInner@http://10.0.2.2:8081/index.bundle?platform=android&dev=true&minify=false&app=com.rn&modulesOnly=false&runModule=true:128667:26
App
RCTView
View
RCTView
View
AppContainer@http://10.0.2.2:8081/index.bundle?platform=android&dev=true&minify=false&app=com.rn&modulesOnly=false&runModule=true:77006:36
rn(RootComponent)@http://10.0.2.2:8081/index.bundle?platform=android&dev=true&minify=false&app=com.rn&modulesOnly=false&runModule=true:83408:28

**Comment by velopert** [comment]
react-navigation 라이브러리 설치 명령어를 한번 더 입력하면 최신버전으로 업데이트 되면서 해결이 될 것 입니다.
